networking/aws_load_balancer_operator/aws-load-balancer-operator-release-notes.adoc

```diff
@@ -13,6 +13,12 @@ These release notes track the development of the AWS Load Balancer Operator in {
 
 For an overview of the AWS Load Balancer Operator, see xref:../../networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc#aws-load-balancer-operator[AWS Load Balancer Operator in {product-title}].
 
++
+[CAUTION]
```

Comment:
The `+` here is unneeded and rendering in the HTML. See [Admonitions](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions) for guidance on the label - we can't use CAUTION, and I suspect NOTE is right, but you might feel like IMPORTANT is warranted based on the definitions.

```suggestion
[NOTE]
```

Reply:
TY